ThingIFSDK/ThingIFSDK/CommandForm.swift

```diff
@@ -45,19 +45,19 @@ public struct CommandForm {
     /**
     Initializer of CommandForm instance.
 
-    - Parameter actions: Array of actions. Must not be empty.
+    - Parameter actions: Array of actions with alias. Must not be empty.
```

Comment:
`Array of actions with alias` better to change to `Array of AliasAction instance `